### PR TITLE
Additional Vanilla Weapons Expanded patch work.

### LINKED ISSUE #617: [Discussion]: Heavy crossbow for VFE-Medieval is kind of bugged?
Just noticed heavy crossbow using the same bolts with normal crossbow but has the same damage, not like short bow and recurve bow using same arrow with different damage in vanilla.

### PR DESCRIPTION
**Pull request by N7Huntsman**
- Add High Roller variants of projectiles, with a 10% chance to double damage.
- Patch Vanilla Weapons Expanded - Tribal (Complete).
- Patch Vanilla Weapons Expanded - Non-lethal (Complete).
- Increases the range of some SMGs.
- Swaps VFE - Security sentry gun from 7.62mm NATO to 5.56mm NATO.
- Adds some additional ammo categories that I felt were generic enough to be useful in the base mod, like rubber bullets and tear gas. 
- Adds damageDefs and injury Hediffs for a tasers to the base mod, since it's may be useful.
- Increases heavy crossbow damage slightly by using stats for a slightly faster bolt.
Closes #617

### CODE REVIEW

**Review by Tostov — CHANGES_REQUESTED** [review]
Shards have too much damage and penetration, somehow outdoing the damage that stabbing someone with a knife directly, when thrown. This is backwards since a knife has the actual mass of someone behind it rather than just its own mass from being tossed.

Firebombs have a duplicate recipe that probably remains from the weapon's thingdef itself most likley.

The taser works acceptably at the moment, but it would be worth considering re-working it to use the muscle spasms system since at the moment it can stun-lock anything regardless of its armor. Which is interesting but maybe a bit too strong.

Otherwise seems good.

File: Patches/Vanilla Furniture Expanded - Security/ThingDefs_Sentry.xml [hunk 1]
```diff
@@ -361,11 +361,8 @@
         <AmmoUser>
           <magazineSize>500</magazineSize>
           <reloadTime>9.2</reloadTime>
-          <ammoSet>AmmoSet_762x51mmNATO</ammoSet>
+          <ammoSet>Ammo_556x45mmNATO_FMJ</ammoSet>
```
Comment (Tostov) [inline Patches/Vanilla Furniture Expanded - Security/ThingDefs_Sentry.xml:364]: Typo'd the ammo in rather than the ammoset. Otherwise all good

**Review by Tostov — CHANGES_REQUESTED** [review]
The ammoset on the sentry is incorrectly defined. Haven't run into anything else during quick tests.

**Review by Tostov — APPROVED** [review]

**Comment by P4V3L-3D0TS** [comment]
what about 40mm bean bag?

**Pull request merged**